gadget/ondisk.go

```diff
@@ -117,28 +117,34 @@ func isCreatedDuringInstall(p *sfdiskPartition, fs *lsblkBlockDevice, sfdiskLabe
 	return false
 }
 
```

Comment:
can you add a couple TODO for when we have some time:

* consider looking into merging LaidOutVolume/Structure OnDiskVolume/Structure